README.md

```diff
@@ -1,116 +1,134 @@
 
-7. Run the development server:
 
-```
-$ python manage.py runserver
-```
+### Running a production server
 
-### Setting up Social Authentication
+Create the files `.envs/.production/.web` and `.envs/.production/.db`, using 
+`.envs/.production/.web.template` and `.envs/.production.db` as reference. 
+We recommend using unique and random keys for the `DJANGO_SECRET_KEY`, 
+`DJANGO_ADMIN_URL`, `POSTGRES_USER` and `POSTGRES_PASSWORD` values.
 
-1. Create a file `config/settings_secret.py` to contain our application secrets. This file is included in the project `.gitignore` file and should not be checked into version control.
+Build the production docker image.
 
-```
-SECRET_KEY = ''
-SOCIAL_AUTH_GITHUB_KEY = ''
-SOCIAL_AUTH_GITHUB_SECRET = ''
-SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = ''
-SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = ''
+```bash
+$ docker-compose --file production.yml --build
 ```
 
-2. A Django `SECRET_KEY` can be generated with the following command:
+Set up the database.
 
-```
-python manage.py shell -c 'from django.core.management import utils; print(utils.get_random_secret_key())'
+```bash
+$ docker-compose --file production.yml run web python manage.py makemigrations
+$ docker-compose --file production.yml run web python manage.py migrate
 ```
 
-3. Register an Oauth2 application on Github: https://github.com/settings/applications/new Enter `http://localhost:8000/complete/github/` as the Authorization callback URL. Once the application is created, enter the Client ID and Client Secret into the `settings_secret.py` file as `SOCIAL_AUTH_GITHUB_KEY` and `SOCIAL_AUTH_GITHUB_SECRET`, respectively.
+Load initial data.
 
-4. Create a new project on the Google Developers Console: https://console.developers.google.com/ . In the Library tab, find and activate the Google+ API (https://console.developers.google.com/apis/library/plus.googleapis.com). Create OAuth client ID Credentials. Select 'Web Application' as the appliction type. Enter `http://localhost:8000/complete/google-oauth2/` as the 'Authorized redirect URI'. Once the application is created, enter the Client ID and Client Secret into the `settings_secret.py` file as `SOCIAL_AUTH_GOOGLE_OAUTH2_KEY` and `SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET`, respectively.
-
-## Viewing the Application
-
- - View on a web browser at: [http://localhost:8000/index.html](http://localhost:8000/index.html)
+```bash
+$ docker-compose --file production.yml run web python manage.py \
+    loaddata sys_admin new_package_template damion_packages
+```
 
- - The Django admin interface can be viewed at: [http://localhost:8000/admin/](http://localhost:8000/admin/). Login with the admin credentials used during the `createsuperuser` step.
+Run the production server.
 
- - The browsable GEEM API can be viewed at: http://localhost:8000/api/
+```bash
+$ docker-compose --file production.yml up --detach
+```
 
+View at [http://localhost:8888/geem/portal.html][3].
```

Comment:
In prelude to production environment setup can you say something about how to install GEEM on production server at [some domain]/geem/ , with NGIX paying attention to port 80 there, and how that translates to a Docker container picking it up (reverse proxy I guess?).  Or is all that taken care of via separate docker server environment?  I'm just thinking of someone having to take on sys admin of geem in our Watson environment.

Reply:
I added a third section below detailing the configurations needed for one's own domain.

There is some extra information about how our watson server is run, which is responsible for mapping port ``80`` to ``8888``, and our ``https`` certificates. I don't think information is well-suited for the public ``README.md``.

Reply:
ok!